go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.2
 require (
 	github.com/Masterminds/sprig/v3 v3.2.3
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
-	github.com/bufbuild/protocompile v0.6.0
+	github.com/bufbuild/protocompile v0.9.0
```

Comment:
Considering you want to optimize makefiles during this PR, why upgrade parts of the go dep?